include/spdlog/tweakme.h

```diff
@@ -103,7 +103,9 @@
 // Uncomment to disable default logger creation.
 // This might save some (very) small initialization time if no default logger is needed.
 //
-// #define SPDLOG_DISABLE_DEFAULT_LOGGER
+#ifdef KPSR_WITH_FREERTOS
```

Comment:
What is this for?

Reply:
The instance of `spdlog` has a default logger of type `sinks::ansicolor_stdout_sink_mt`, that contains an `std::unordered_map` of colors. It could be also refactored to use an `std::map`, but this define just disables all of it.